Application.mk

```diff
@@ -62,7 +62,10 @@ ifeq ($(BUILD_MODULE),y)
 endif
 
 SUFFIX = $(subst $(DELIM),.,$(CWD))
+
+ifeq ($(CONFIG_WINDOWS_NATIVE),)
 PROGNAME := $(shell echo $(PROGNAME))
```

Comment:
can we change `$(subst ",,$(PROGNAME))` to avoid ifeq

Reply:
tested both on linux and windows native